docs/wiki/Deploying-ALZ-ZTNetwork.md

```diff
@@ -0,0 +1,49 @@
+## Azure landing zone portal accelerator deployment with Zero Trust network principles
+
+This section describes how to deploy the Azure landing zone portal accelerator with a jump start on Zero Trust Networking Principles for Azure landing zones. For more information on Zero Trust security model and principles visit [Embrace proactive security with Zero Trust](/security/business/zero-trust). Let's review the [Zero Trust aligned networking](/security/zero-trust/deploy/networks) configurations in the [Azure landing zone portal accelerator](/azure/cloud-adoption-framework/ready/landing-zone/#azure-landing-zone-portal-accelerator).
+
```

Comment:
Do we need to add a note here saying that the steps below are not a complete end-to-end experience of walking through the portal, just for clarity to prevent issues being raised for something we didnt intend to create?

Reply:
Great call out Jack. I've updated to make it clear we are focusing on zero trust networking and that there are additional guides in the navigation on the right.